lib/std/tempfiles.nim

```diff
@@ -103,46 +116,61 @@ proc genTempPath*(prefix, suffix: string, dir = ""): string =
   let dir = getTempDirImpl(dir)
   result = dir / (prefix & randomPathName(nimTempPathLength) & suffix)
 
-proc createTempFile*(prefix, suffix: string, dir = ""): tuple[fd: File, path: string] =
-  ## Creates a new temporary file in the directory `dir`.
+proc createTempFile*(prefix, suffix: string, dir = ""): tuple[cfile: File, path: string] =
+  ## Creates a new temporary file in the directory `dir`, which must exist
+  ## (empty `dir` will resolve to `getTempDir()`).
```

Comment:
Hm, is there a better way to phrase this? Typically documentation goes something like:
```
Brief summary

Functionality explanation

Argument explanation
```

Reply:
i think the way i wrote it was clear enough but I've followed your suggestion anyway